bindings/java/build.gradle

```
@@ -0,0 +1,58 @@
+repositories {
+    google()
+    jcenter()
+}
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.0.0'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        minSdkVersion 28
+        targetSdkVersion 29
+
+        externalNativeBuild {
+            cmake {
+                // This is being checked in for now with the hard-coded linux
+                // paths due to some complications with configuring
+                // cross-compilation.
+                arguments '-DIREE_HOST_C_COMPILER=/usr/bin/clang',
+                        '-DIREE_HOST_CXX_COMPILER=/usr/bin/clang++',
+                        '-DIREE_BUILD_COMPILER=OFF',
+                        '-DIREE_ENABLE_MLIR=OFF',
+                        '-DIREE_BUILD_TESTS=OFF',
+                        '-DIREE_BUILD_SAMPLES=OFF',
+                        '-DIREE_BUILD_JAVA_BINDINGS=ON',
+                        '-DIREE_BUILD_PYTHON_BINDINGS=OFF',
+                        '-DIREE_HAL_DRIVERS_TO_BUILD=VMLA',
+                        '-DIREE_TARGET_BACKENDS_TO_BUILD=VMLA'
```

nit: we could organize this list of arguments to make it more clear why each option is used

assuming inline comments are allowed:
```suggestion
                arguments 
                    '-DIREE_BUILD_JAVA_BINDINGS=ON',
                    '-DIREE_HAL_DRIVERS_TO_BUILD=VMLA',
                       
                    // Disable all but the runtime components needed for the java bindings.
                    '-DIREE_BUILD_COMPILER=OFF',
                    '-DIREE_ENABLE_MLIR=OFF',
                    '-DIREE_BUILD_TESTS=OFF',
                    '-DIREE_BUILD_SAMPLES=OFF',                        
                    '-DIREE_BUILD_PYTHON_BINDINGS=OFF',
 
                    // This is being checked in for now with the hard-coded linux
                    // paths due to some complications with configuring
                    // cross-compilation.
                    '-DIREE_HOST_C_COMPILER=/usr/bin/clang',
                    '-DIREE_HOST_CXX_COMPILER=/usr/bin/clang++',
```

You should also be able to remove `TARGET_BACKENDS_TO_BUILD` as that is for the compiler:
```
                        -DIREE_TARGET_BACKENDS_TO_BUILD=VMLA
```

---

btw, just filed https://github.com/google/iree/issues/4520 for `IREE_HOST_C_COMPILER`/`IREE_HOST_CXX_COMPILER` and other cross compilation messiness

---

Done, this looks much nicer now!